cmd/podman/machine/init.go

```diff
@@ -127,6 +127,9 @@ func init() {
 	userModeNetFlagName := "user-mode-networking"
 	flags.BoolVar(&initOptionalFlags.UserModeNetworking, userModeNetFlagName, false,
 		"Whether this machine should use user-mode networking, routing traffic through a host user-space process")
+	
+	rosettaFlagName := "rosetta"
+	flags.BoolVar(&initOpts.Rosetta, rosettaFlagName, true, "Whether this machine should use Rosetta, running x86_64 Linux binaries under ARM Linux on Apple silicon")
```

Comment:
first, thanks for the PR and pushing this topic.  We had not intended to contemplate this work for a while yet, so you are pushing us to deal with this earlier than we wanted.  As such, a bunch of my comments are initial reactions to how you choose to implement this. 

I had not intended to expose a flag only useful on a singular platform in the machine CLI.  There is precedent for this however with usermode networking.  But this could easily drift into optimization flavors for all kinds of different platforms.

Perhaps this function should only be exposed in some way in containers common?  Maybe even usermode networking too ?

if we end up merging this, i think we consider if this should be `false` for the initial 5.0 release. we generally take cautious approaches to this.

Reply:
fixed. removed `--rosetta` option.